app/Eloquent/Profile.php

```diff
@@ -20,8 +24,27 @@ class Profile extends Model
 {
     protected $primaryKey = "user_id";
 
+    protected $fillable = ["public_name", "avatar_path", "birthday"];
+
```

Comment:
This empty line is unnecessary. Look at this: https://www.php-fig.org/psr/psr-12/#43-properties-and-constants